packages/sanity/src/core/form/inputs/PortableText/Compositor.tsx

```diff
@@ -395,34 +396,43 @@ export function Compositor(props: Omit<InputProps, 'schemaType' | 'arrayFunction
 
   const editorNode = useMemo(
     () => (
-      <Editor
-        ariaDescribedBy={ariaDescribedBy}
-        elementRef={elementRef}
-        initialSelection={initialSelection}
-        hideToolbar={hideToolbar}
-        hotkeys={editorHotkeys}
-        isActive={isActive}
-        isFullscreen={isFullscreen}
-        onItemOpen={onItemOpen}
-        onCopy={onCopy}
-        onPaste={onPaste}
-        onToggleFullscreen={handleToggleFullscreen}
-        path={path}
-        rangeDecorations={rangeDecorations}
-        readOnly={readOnly}
-        renderAnnotation={editorRenderAnnotation}
-        renderBlock={editorRenderBlock}
-        renderChild={editorRenderChild}
-        renderEditable={renderEditable}
-        setPortalElement={setPortalElement}
-        scrollElement={scrollElement}
-        setScrollElement={setScrollElement}
-      />
+      <UploadTargetCard
+        types={editor.schemaTypes.portableText.of}
+        resolveUploader={props.resolveUploader}
+        onUpload={props.onUpload}
+        {...props.elementProps}
```

Comment:
Do we need to set the elementProps here?

Reply:
They are passed like this down to whatever component the `fileTarget` HOC is wrapping. Not sure if this is needed, but i'd rather not touch it since we do this multiple places.

Reply:
Removed.